TrackPropagation/RungeKutta/src/RKAdaptiveSolver.icc

```diff
@@ -40,11 +39,8 @@ typename RKAdaptiveSolver<T, StepWithPrec, N>::Vector RKAdaptiveSolver<T, StepWi
     float acc = tryStep.second;
     //assert(eps>0);
     // assert(acc>=0);
-    nsteps++;
     if (acc < eps || std::abs(stepSize) < std::abs(remainigStep) * 0.1f) {
       if (std::abs(remainigStep - stepSize) < 0.5f * eps) {
-        //	if (verbose()) std::cout << "Accuracy reached, and full step taken in "
```

Comment:
maybe some form of ```#ifdef``` would be more appropriate than just removing those debug statements

Reply:
Sure, I already applied your suggestion. Thanks @VinInn!